server/validators/stepTargetDateValidator.ts

```diff
@@ -0,0 +1,17 @@
+import moment from 'moment'
+
+export default function validateStepTargetDate(targetDate?: Date): Array<string> {
+  const errors: Array<string> = []
+
+  if (!targetDate) {
+    return errors
+  }
```

Comment:
The `targetDate` is optional. Not sure if this is the best way to do it in typescript.

Reply:
Thats OK, its basically saying if the targetDate value is "falsy" (null or undefined - they are 2 different values!). then return errors, which will be an empty array - so all good 👍